.github/workflows/project-automation.yml

```
@@ -8,10 +8,12 @@ on:
 jobs:
   add-to-project-board:
     name: Add issue to project
-    if: github.repository != 'FlowFuse/website'
+    if: | 
+      github.repository != 'FlowFuse/website' ||
```

```suggestion
      github.repository != 'FlowFuse/website' &&
```

Should this not be `&&` logic? Because the `@flowfuse/website` will be `true` on the second logic test and therefore run.

---

Absolutely, you are right, thanks!